modules/gapi/samples/pipeline_modeling_tool/utils.hpp

```diff
@@ -23,11 +37,17 @@ inline void generateRandom(cv::Mat& out) {
             cv::randu(out, 0.f, 1.f);
             break;
         case CV_16F: {
-            cv::Mat fp32_mat(out.size(), CV_MAKETYPE(CV_32F, out.channels()));
```

Comment:
If `out` is `ND` (honestly it's always `ND`, since we create `cv::Mat's` from `std::vector`) calling `size` method cause exception